recipes/libogg/meta.yaml

```diff
@@ -0,0 +1,41 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "1.3.2" %}
+
+package:
+    name: libogg
+    version: {{ version }}
+
+source:
+    fn: libogg-{{ version }}.tar.gz
+    url: http://downloads.xiph.org/releases/ogg/libogg-{{ version }}.tar.gz
+    # md5 is OK but sha256 is desirable.
+    sha256: e19ee34711d7af328cb26287f4137e70630e7261b17cbe3cd41011d73a654692
+
+build:
+    number: 0
+    # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+    script: true
```

Comment:
You can remove this. But (given the script) we likely want `skip: True  # [win]`